third_party/move/move-compiler-v2/tests/checking/abilities/v1/ability_constraint_structs_invalid.exp

```diff
@@ -0,0 +1,114 @@
+
```

Comment:
Lines 29 on previously (v1) have errors involving phantom type parameters that are missing now.